tuist/tuist
# Add support for TUIST_CLOUD_URL to override the default Cloud URL

### Short description 📝

This PR adds a support for `TUIST_CLOUD_URL` that allows users to override the default URL (https://cloud.tuist.io)

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

Run `TUIST_CLOUD_URL="http://127.0.0.1:3000" swift run tuist cloud auth`. This should open an auth page of a local server (regardless if you have one running locally).

### Contributor checklist ✅


### Reviewer checklist ✅
